src/app/payments/helpers.ts

```diff
@@ -164,6 +163,26 @@ const checkLimitsBase = async ({
     (vol): vol is TxBaseVolumeAmount<WalletCurrency> => true,
   )
 
+  return walletVolumes
+}
+
+const checkLimitsBase = async ({
```

Comment:
Can we avoid the word `Base`? This is not OO - hence it's not a precise term. Can easily be misinterpreted.

Reply:
Fixed: https://github.com/GaloyMoney/galoy/pull/1953/commits/45dd3d0399cc8aa37a5d7036993c864903c5e7ed